### PR TITLE
Fixed Reduced Service prompt not being string

### LINKED ISSUE #4083: Reduced Service doesn't show credits
Hey. An issue with one of the new Downfall cards, Reduced Service.

When the card is rezzed, it comes up with buttons to select the amount of counters you want to put on the card, but all buttons are blank, so it's difficult to know which is the correct button to press. 

Thanks.

### PR DESCRIPTION
**Pull request by presheaf**
Closes #4083 .

### CODE REVIEW

**Comment by presheaf** [comment]
Oh hang on, I got a bit too eager here.

**Comment by NoahTheDuke** [comment]
haha I just did the same thing. I wonder why your test didn't work!

**Comment by NoahTheDuke** [comment]
Oh, didn't convert back to int when it's `target`.

**Comment by presheaf** [comment]
Right on!

**Comment by presheaf** [comment]
And I remember now that this was why I made the let binding originally. :P I just didn't finish the thought at all.

**Comment by presheaf** [comment]
should I just close this PR?

**Comment by NoahTheDuke** [comment]
Nah this is fine! I’ll close mine

File: src/clj/game/cards/upgrades.clj [hunk 1]
```diff
@@ -1056,13 +1056,14 @@
...
                                                :prompt "How many credits to spend?"
                                                :effect (req (clear-wait-prompt state :runner)
-                                                            (deduct state :corp [:credit target])
-                                                            (add-counter state :corp card :power target)
-                                                            (system-msg state :corp (str "place " target " power counters on Reduced Service"))
-                                                            (effect-completed state side eid))}
+                                                            (let [spent (str->int target)]
+                                                              (deduct state :corp [:credit spent])
+                                                              (add-counter state :corp card :power spent)
+                                                              (system-msg state :corp (str "place " spent " power counters on Reduced Service"))
```
Comment (NoahTheDuke) [inline src/clj/game/cards/upgrades.clj]:
```suggestion
                                                              (system-msg state :corp (str "places " spent " power counters on Reduced Service"))
```

Comment (Saintis) [reply src/clj/game/cards/upgrades.clj]: You should use quantify with `spent` to correctly add s to power counters here.

Comment (NoahTheDuke) [reply src/clj/game/cards/upgrades.clj]: Good call

Comment (NoahTheDuke) [inline src/clj/game/cards/upgrades.clj]:
```suggestion
                                                              (system-msg state :corp (str "places " (quantify spent "power counter") on Reduced Service"))
```

**Pull request merged**